packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -150,6 +146,23 @@ def _setup_logging():
     logging.getLogger("requests").setLevel(logging.WARNING)
 
 
+@contextlib.contextmanager
```

Comment:
`monkeypatch` has support for setting / removing environment variables. Can we just use that?

http://doc.pytest.org/en/latest/monkeypatch.html#_pytest.monkeypatch.monkeypatch.setenv